components/core/tests/test-ir_encoding_methods.cpp

```diff
@@ -1177,13 +1215,13 @@ TEMPLATE_TEST_CASE(
 
 // NOLINTNEXTLINE(readability-function-cognitive-complexity)
 TEMPLATE_TEST_CASE(
-        "ffi_ir_stream_Serializer_serialize_msgpack",
-        "[clp][ffi][ir_stream][Serializer]",
+        "ffi_ir_stream_kv_pair_log_events_serder",
```

Comment:
```suggestion
        "ffi_ir_stream_kv_pair_log_events_serde",
```

The abbreviation is "serde," right? As in "serialize, deserialize" rather than "serialize, derserialize".

Reply:
True true, didn't realize there's a difference between "serde" and "serder"